sample/WebHook-Generic-CSharp/run.csx

```diff
@@ -2,9 +2,14 @@
 
 using Newtonsoft.Json;
 using Newtonsoft.Json.Linq;
+using System.Security.Claims;
 
-public static JObject Run(Payload payload, string action)
+public static JObject Run(Payload payload, string action, ClaimsIdentity identity, TraceWriter log)
```

Comment:
Another instance where I wonder if we should leave the basic sample alone, and have a separate auth sample.
Also, webhooks generally don't include a user identity, so I'm not sure how valid this scenario is.